### DISCUSSION TITLE
Request: Release the multipart parser as a separate package

### DISCUSSION THREAD

**Discussion by Goldziher** [abersheeran/baize#48] [General]
Hi there!

I am one of the maintainers of [Starlite](https://github.com/starlite-api/starlite). We received a request to replace our usage of [python-multipart](https://github.com/andrew-d/python-multipart) with your implementation. I looked at the implementation, it does appear like a solid foundation to build on. I was wondering whether you'd consider releasing the multipart parser you wrote as  a separate package? This would be very helpful for us since it will allow us to minimize the dependencies.

Cheers!

BTW, you project looks very interesting. I'd be happy if you'd like to contribute to Starlite as well 😉

**Comment by abersheeran** [comment]
I'm not interested in disassembling multipart parts into a separate package, baize aims to be a general toolkit similar to werkzeug, not a complete, easy-to-use web framework. Several web frameworks have been built on top of it, and the API provided by baize can basically be considered stable (I'm waiting for the perfection of mypyc, and then release the 1.0.0 version). 

You can use it directly to parse data in multipart format.

**Reply by Goldziher** [reply]
ok, thanks. We wont be using it.
